net/gatling/files/gatling.conf

```diff
@@ -0,0 +1,28 @@
+
+config gatling 'v4'
+	option listen_http '0.0.0.0:80'
+	option virtual_hosting on
```

Comment:
I would suggest to disable virtual hosts by default. The first thing I have tried is "http://192.168.1.1". But this won't work with virtual hosts.

Reply:
This should work, unless the 192.168.1.1 is handled by gatling.

For vhosts to work properly, gatling expects a correspondant directory in the webroot, in wich it will `cd`. For your example given, there must be a dir `192.168.1.1:80` with an `index.html` located in it.

Virtual hosts can get disabled in the config file too. Although I see it as a major advantage over other webservers, I'd be fine with setting it default off.

Reply:
Please set virtual hosts to off in the config file. This makes the first user experience much nicer (it was for me).

Reply:
addressed in the current commits version.